src/main/java/log_in/use_case/LogInInteractor.java

```diff
@@ -2,26 +2,30 @@
 
 import entities.CurrentUser;
 import entities.User;
-import favourite.use_case.UserFactory;
 import log_in.use_case.exceptions.UserException;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class LogInInteractor implements LogInInputBoundary {
     private final LogInDsGateway access;
-    private final UserFactory userFactory;
+
 
     private final LogInOutputBoundary outputBoundary;
 
+    private final UserFactory userFactory;
+
 
     /**
      * Initialize a LogInInteractor
      *
      * @param access to retrieve stuff from the database
      * @param outputBoundary contains the result of this use case
      */
-    public LogInInteractor(LogInDsGateway access, LogInOutputBoundary outputBoundary, UserFactory userFactory){
+    public LogInInteractor(LogInDsGateway access, LogInOutputBoundary outputBoundary, UserFactory factory){
         this.access = access;
         this.outputBoundary = outputBoundary;
-        this.userFactory = userFactory;
+        this.userFactory = new UserFactory();
```

Comment:
A factory is being passed as a parameter, do we want to use that or do we want to initialize a new one here?

Reply:
Yeah, forgot to assign it thanks